unittests/testpublicassetsread.py

```diff
@@ -25,30 +32,73 @@
 
 LOGGER = getLogger(__name__)
 
+URL = "https://app.rkvst.io"
+ASSET_ID = f"{URL}/{ROOT}/{ASSETS_SUBPATH}/public{IDENTITY}"
+SUBPATH = f"{ASSETS_SUBPATH}/public{IDENTITY}"
+
+ASSET_ID_NO_SUBPATH = f"{URL}/{ROOT}/public{IDENTITY}"
+NO_SUBPATH = f"public{IDENTITY}"
+
 
 class TestPublicAssetsRead(TestPublicAssetsBase):
     """
-    Test Archivist Public Assets methods
+    Test Archivist Assets methods
     """
 
-    def test_public_assets_read_with_out_primary_image(self):
+    def test_publicassets_read(self):
         """
-        Test public asset reading
+        Test asset reading
         """
-        with mock.patch.object(self.arch._session, "get") as mock_get:
+        with mock.patch.object(self.public.session, "get") as mock_get:
             mock_get.return_value = MockResponse(200, **RESPONSE_NO_ATTACHMENTS)
 
-            asset = self.arch.publicassets.read(IDENTITY)
+            asset = self.public.assets.read(ASSET_ID)
```

Comment:
Is this actually supposed to work? I thought we'd dropped versions (ASSET_SUBPATH) from calls to GET public assets. Check with an API architect for what's supposed to work after the public assets endpoint is implemented.

Reply:
thait is wrong

Reply:
self.public is an ArchivistPublic() endpoint that is instantiated in the setUp() method of the class.

ASSET_ID is the public url os the asset - in this PR it includes the SUBPATH because the publicutl stuff was not complete when this was written.

Reply:
new PR merged to fix this...